### PR TITLE
Add example keywords

### LINKED ISSUE #671: Update examples to use keywords for parameters
The examples have magic numbers for the values of some of the functions. It would be clearer if we updated these examples to use keywords, so customers know the values that we are setting.

### PR DESCRIPTION
**Pull request by Fladolcetta**
[x ] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

- Adds keywords to the examples for improved example understanding.
- Renamed args.name to args.resource_name for consistency
- Adds method to simulate devices for eventual system testing (option string on most drivers, topology and simulate flag for NI-SWITCH).
- Changed NI-FGEN examples to use a time value instead of waiting for input in the console. This is needed for eventual system testing.
- Removed rate from FGEN Arb example as this is not used. 
- Fixed name of enum in NI-SWITCH example.

### List issues fixed by this Pull Request below, if any.

* Fix #671 

### What testing has been done?

Ran tox.
Ran examples

### CODE REVIEW

File: src/nidcpower/examples/nidcpower_advanced_sequence.py [hunk 1]
```diff
@@ -12,10 +12,11 @@
 parser.add_argument('-vf', '--voltage_final', default=3.0, type=float, help='Voltage level at which sweep ends')
 parser.add_argument('-c0', '--current_start', default=100e-6, type=float, help='Current level at which sweep starts')
 parser.add_argument('-cf', '--current_final', default=300e-6, type=float, help='Current level at which sweep ends')
+parser.add_argument('-op', '--option', default='', type=str, help='Option String')
```
[the remark this answers is not in the document]
Comment (Fladolcetta) [reply src/nidcpower/examples/nidcpower_advanced_sequence.py]: Fixed all but the single character one. The reason is that Fgen uses -o for offset. I wanted the option string to be consistent on all examples so I picked -op.

File: src/nidcpower/examples/nidcpower_advanced_sequence.py [hunk 2]
```diff
@@ -30,13 +31,13 @@ def create_sweep(begin_value, end_value, number_of_steps):
     return sweep
 
 
-with nidcpower.Session(args.name, channels=args.channels) as session:
+with nidcpower.Session(resource_name=args.name, channels=args.channels, option_string=args.option) as session:
```
[the remark this answers is not in the document]
Comment (Fladolcetta) [reply src/nidcpower/examples/nidcpower_advanced_sequence.py]: fixed

File: src/nifgen/examples/nifgen_standard_function.py [hunk 3]
```diff
@@ -2,22 +2,22 @@
...
 supported_waveforms = list(nifgen.Waveform.__members__.keys())[:-1]  # no support for user-defined waveforms in example
 parser = argparse.ArgumentParser(description='Generates the standard function.', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-parser.add_argument('-n', '--name', default='PXI1Slot2', help='Resource name of a National Instruments Function Generator')
+parser.add_argument('-n', '--resource_name', default='PXI1Slot2', help='Resource name of a National Instruments Function Generator')
 parser.add_argument('-w', '--waveform', default=supported_waveforms[0], choices=supported_waveforms, type=str.upper, help='Standard waveform')
 parser.add_argument('-f', '--frequency', default=1000, type=float, help='Frequency (Hz)')
 parser.add_argument('-a', '--amplitude', default=1.0, type=float, help='Amplitude (Vpk-pk)')
 parser.add_argument('-o', '--offset', default=0.0, type=float, help='DC Offset (V)')
 parser.add_argument('-p', '--phase', default=0.0, type=float, help='Start Phase (deg)')
+parser.add_argument('-t', '--time', default=5, type=float, help='Generation Time')
```
Comment (marcoskirsch) [inline src/nifgen/examples/nifgen_standard_function.py]: if type is float, then default should be typed as float: 5.0
help should not be title case, and it should indicate the units.

File: src/nifgen/examples/nifgen_standard_function.py [hunk 4]
```diff
@@ -10,9 +10,9 @@
 parser.add_argument('-w', '--waveform', default=supported_waveforms[0], choices=supported_waveforms, type=str.upper, help='Standard waveform')
 parser.add_argument('-f', '--frequency', default=1000, type=float, help='Frequency (Hz)')
 parser.add_argument('-a', '--amplitude', default=1.0, type=float, help='Amplitude (Vpk-pk)')
-parser.add_argument('-o', '--offset', default=0.0, type=float, help='DC Offset (V)')
-parser.add_argument('-p', '--phase', default=0.0, type=float, help='Start Phase (deg)')
-parser.add_argument('-t', '--time', default=5, type=float, help='Generation Time')
+parser.add_argument('-o', '--offset', default=0.0, type=float, help='DC offset (V)')
+parser.add_argument('-p', '--phase', default=0.0, type=float, help='Start phase (deg)')
```
Comment (marcoskirsch) [inline src/nifgen/examples/nifgen_standard_function.py]: But are these degrees Celsius or Fahrenheit?

(just kidding)

**Pull request merged**